renovate.json

```diff
@@ -1,4 +1,10 @@
 {
   "extends": ["config:base", ":preserveSemverRanges"],
-  "schedule": "before 3am on Monday"
+  "schedule": "before 3am on Monday",
+  "packageRules": [
+    {
+      "matchPackageNames": ["react-dom", "react-server-dom-webpack"],
+      "enabled": false
```

Comment:
Would it make sense to leverage https://docs.renovatebot.com/configuration-options/#allowedversions instead?
So we can prevent updates for unwanted versions, but still potentially get future updates

Reply:
I’m open to better ways, I don’t know renovate.
But react with RSC is on 0.0.0-experimental. All other versions do not have RSC and break

Reply:
Know of a way to get around this? Or is it fine?